src/ui/sso/containers/consent.tsx

```diff
@@ -7,14 +7,14 @@ import { withLoading, withErrorScreen } from 'src/actions/modifiers'
 import {
   CredentialTypeSummary,
   CredentialVerificationSummary,
-  CredentialRequestFlowState,
 } from 'src/lib/interactionManager/types'
 import { NavigationScreenProp, NavigationState } from 'react-navigation'
 import { InteractionSummary } from '../../../lib/interactionManager/types'
 
 interface CredentialRequestNavigationParams {
   interactionId: string
   interactionSummary: InteractionSummary
+  availableCredentials: Array<CredentialTypeSummary>
```

Comment:
not the prettiest way to add this when there is also 'interactionSummary", but I think the UI convenience concerns shouldn't influence the state type of the interaction